common/logisticspipes/utils/InventoryUtilFactory.java

```diff
@@ -21,6 +21,8 @@
 import net.minecraftforge.common.capabilities.ICapabilityProvider;
 import net.minecraftforge.items.CapabilityItemHandler;
 
+import appeng.api.storage.IMEInventoryHandler;
```

Comment:
Since nothing else is changed in this class, is this an unused import? If so, please remove